ssl/statem/statem_lib.c

```diff
@@ -614,13 +614,6 @@ MSG_PROCESS_RETURN tls_process_key_update(SSL *s, PACKET *pkt)
 {
     unsigned int updatetype;
 
-    s->key_update_count++;
-    if (s->key_update_count > MAX_KEY_UPDATE_MESSAGES) {
```

Comment:
The MAX_KEY_UPDATE_MESSAGES define probably still exists

Reply:
Good point. I removed the define too.